app/src/layouts/common/appSidebar/helpAndService/FAQcontent/FAQContent.jsx

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright 2024 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { FormattedMessage, useIntl } from 'react-intl';
+import OpenIcon from 'common/img/open-in-new-tab-inline.svg';
+import React from 'react';
+import { referenceDictionary, uniqueId } from 'common/utils';
+import { showModalAction } from 'controllers/modal';
+import { useDispatch, useSelector } from 'react-redux';
+import { userIdSelector } from 'controllers/user';
+import { setFAQOpenStatusTrue } from 'controllers/log/storageUtils';
+import { messages } from 'layouts/common/appSidebar/helpAndService/messages';
+import { LinkItem } from 'layouts/common/appSidebar/helpAndService/linkItem/linkItem';
```

Comment:
Please use relative paths.
The `LinkItem` and `messages` for now are developed to be used inside `helpAndService` only.